pages/release-notes.mdx

```diff
@@ -7,14 +7,17 @@ import { Steps } from "nextra/components"
 
 <Steps>
 <Step>
-## November 2nd 2023 - v0.73.0
+## 2 November 2023 - v0.73.0
 
-#### 🚨 Generate seeds change
+#### 🚨 Generate seeds breaking change 🚨
```

Comment:
Let's use a single leading emoji, no need to repeat it at the end.

Reply:
Agreed